packages/gatsby-remark-prismjs/src/directives.js

```diff
@@ -154,7 +154,9 @@ module.exports = function highlightLineRange(code, highlights = []) {
       })
       .map(line => {
         if (line.highlight) {
-          line.code = highlightWrap(line.code)
+          line.code
```

Comment:
better check for empty string or null? Otherwise a line with number `0` and text `false` would be also exchanged?

Reply:
Prism highlights from 1 and up. You cannot highlight line 0 as there is no such line.

Reply:
Snapshot tests are failing as I am injecting non breakable space into empty lines (&nbsp;) so these tests should just get the snapshots updated.